model/UpdateAccountRequest.ts

```diff
@@ -7,3 +8,11 @@ export interface UpdateAccountRequest {
 	friendlyName?: string
 	fundingLimit?: FundingLimitRequest
 }
+export namespace UpdateAccountRequest {
+	export const type = isly.object<UpdateAccountRequest>({
+		state: isly.string(),
```

Comment:
state: isly.string([...]),

If AccountState is used by other types then it would be good to make it's own type.